components/domains/registerV2.tsx

```diff
@@ -200,6 +201,20 @@ const RegisterV2: FunctionComponent<RegisterV2Props> = ({ domain, groups }) => {
       calls.push(registrationCalls.addressToDomain(encodedDomain));
     }
 
+    // If the user has toggled autorenewal
+    if (renewalBox) {
+      const limitPrice = salesTaxAmount
```

Comment:
@Th0rgal are you sure it works like this ? Can we follow this after ? If yes then no problem